tests/ecosystem/deployment/test_deployment.py

```diff
@@ -78,3 +85,32 @@ def test_deployment(pvc_factory, pod_factory):
 
     if teardown:
         log.info("Cluster will be destroyed during teardown part of this test.")
+
+
+@azure_platform_required
```

Comment:
I thin that this decorator is not necessary here as it's not test anymore.
Also suggests to move outside of this test file to:

ocs_ci/utility/azure_utils.py